multiversx_sdk_cli/cli_wallet.py

```diff
@@ -95,7 +95,7 @@ def new_wallet(args: Any):
 
     if args.pem:
         pem_file = prepare_file(args.output_path, ".pem")
-        address = Address(pubkey)
+        address = Address(pubkey.buffer)
```

Comment:
Can pubkey be `nil` so you would need to check before calling `.buffer` ?

Reply:
It cannot, it's generated here:

```
pubkey = secret_key.generate_public_key()
```